### PR TITLE
Add job.duration in web ui

### PR DESCRIPTION
**Pull request by yp05327**
image
image

Maybe we can change the location of it.

### CODE REVIEW

**Review by lunny — APPROVED** [review]

**Comment by lunny** [comment]
Hm, but should that time increment when the job is still running?

**Comment by yp05327** [comment]
@lunny 
Do you mean this?
6llng-cgp0g

**Review by Zettat123 — APPROVED** [review]

**Review by wolfogre — APPROVED** [review]

File: web_src/js/components/RepoActionView.vue [hunk 1]
```diff
@@ -32,6 +32,7 @@
                 <ActionRunStatus :status="job.status"/>
                 <span class="ui text gt-mx-3">{{ job.name }}</span>
               </a>
+              <span class="step-summary-dur">{{ job.duration }}</span>
```
Comment (silverwind) [inline web_src/js/components/RepoActionView.vue]:
```suggestion
              <span class="step-summary-duration">{{ job.duration }}</span>
```

No need for obscure abbreviations, also change other occurences in the file please.

Comment (yp05327) [reply web_src/js/components/RepoActionView.vue]: Thanks for your suggestion.
Done.

**Review by silverwind — APPROVED** [review]

**Comment by jolheiser** [comment]
🎺 🤖

**Pull request merged**